src/domain/orders/details/create-fulfillment/index.tsx

```diff
@@ -60,6 +101,24 @@ const CreateFulfillmentModal: React.FC<CreateFulfillmentModalProps> = ({
   const notification = useNotification()
 
   const createFulfillment = () => {
+    if (
+      isFeatureEnabled("inventoryService") &&
```

Comment:
suggestion(non blocking): We should think of centralizing those static values in a single place maybe? As a user, I might have no idea where it comes from. Plus it would help a user to reuse them without having to search for all possible values everywhere. wdyt?